Source/WebKit/Shared/API/APIData.h

```diff
@@ -76,6 +81,17 @@ class Data : public ObjectImpl<API::Object::Type::Data> {
     static Ref<Data> createWithoutCopying(RetainPtr<NSData>);
 #endif
 
+#if PLATFORM(GTK)
+    static Ref<Data> createWithoutCopying(GRefPtr<GBytes> bytes)
+    {
+        gsize size;
+        const void* rawData = g_bytes_get_data(bytes.get(), &size);
+
+        auto data = unsafeMakeSpan(static_cast<const uint8_t*>(rawData), size);
+        return createWithoutCopying(data, [bytes = WTFMove(bytes)] () { });
```

Comment:
We have a `span(const GRefPtr<GBytes&>)` helper in `<wtf/glib/GSpanExtras.h>` which would be perfect to use here:

```suggestion
        return createWithoutCopying(span(bytes), [bytes = WTFMove(bytes)] () { });
```